### ISSUE TITLE
Distributed training with tensorflow doesn't work 

### ISSUE THREAD

**Issue by yu-iskw** [polyaxon/polyaxon#387]
### Describe the bug
I was not able to run distributed training with tensorflow under polyaxon 0.4.0 on google cloud, even though the same code was able to run with polyaxon 0.3.9.
I got the following error about `TypeError` in `tensorflow_spawner.py`.

#### Error message
```
Traceback (most recent call last): File "/polyaxon/polyaxon/scheduler/experiment_scheduler.py", line 493, in start_experiment use_sidecar=True) File "/polyaxon/polyaxon/scheduler/spawners/experiment_spawner.py", line 99, in __init__ self.job_uuids = self.create_job_uuids() File "/polyaxon/polyaxon/scheduler/spawners/tensorflow_spawner.py", line 108, in create_job_uuids uuid.uuid4().hex for _ in self.get_n_pods(task_type=TaskType.WORKER)] TypeError: 'int' object is not iterable
```

#### Experiment status
<img width="924" alt="screen shot 2019-02-26 at 10 57 57 am" src="https://user-images.githubusercontent.com/1523515/53438755-6782cd00-39b5-11e9-906c-feef788e67e5.png">

### To Reproduce
```
---
version: 1

kind: experiment

environment:
  node_selector:
    polyaxon: experiments-gpu-t4
  resources:
    cpu:
      requests: 4
      limits: 4
    memory:
      requests: 15000
      limits: 15000
    gpu:
      requests: 1
      limits: 1
  tolerations:
    - key: nvidia.com/gpu
      operator: Equal
      value: present
      effect: "NoSchedule"

  tensorflow:
    n_workers: 1
    n_ps: 1

    default_worker:
      node_selector:
        polyaxon: experiments-gpu-t4
      resources:
        cpu:
          requests: 4
          limits: 4
        memory:
          requests: 15000
          limits: 15000
        gpu:
          request: 1
          limits: 1
      tolerations:
        - key: nvidia.com/gpu
          operator: Equal
          value: present
          effect: "NoSchedule"
    default_ps:
      node_selector:
        polyaxon: experiments-gpu-t4
      resources:
        cpu:
          requests: 4
          limits: 4
        memory:
          requests: 5000
          limits: 5000
        gpu:
          requests: 0
          limits: 1
      tolerations:
        - key: nvidia.com/gpu
          operator: Equal
          value: present
          effect: "NoSchedule"

build:
  image: tensorflow/tensorflow:1.10.1-gpu
  build_steps:
    - pip install --no-cache-dir -r requirements.txt

run:
  cmd:  python src/mnist.py
```

### Expected behavior
The distributed training should run.

### Environment
- polyaxon: 0.4.0
- Google Kuebernetes Engine: v1.11.6-gke.11

**Comment by mouradmourafiq** [comment]
related to distributed pytorch error #386

**Comment by yu-iskw** [comment]
I got it. I look forward to 0.4.1!

**Comment by mouradmourafiq** [comment]
Closing this, a patch will be released in v0.4.1 today or tomorrow.

**Comment by yu-iskw** [comment]
Awesome! Thank you very much for fixing it quickly!!
